src/libcore/ops.rs

```diff
@@ -445,7 +445,10 @@ rem_impl! { usize u8 u16 u32 u64 isize i8 i16 i32 i64 }
 impl Rem for f32 {
     type Output = f32;
 
-    // see notes in `core::f32::Float::floor`
+    // The builtin f32 frem doesn't compile when targeting
+    // msvc; use f64 rem instead.
+    // FIXME: This is a bug; trans should be fixed instead
+    // of working around it here.
```

Comment:
It's not explicitly using an intrinsic but it's the same idea, LLVM is lowering this operation to a function call which isn't implemented on MSVC (explaining the `#[cfg]` block). 

The current comment is somewhat odd with a reference to "frem" (requires knowledge of compiler internals) and has a FIXME which isn't actionable (needs to be altered in upstream LLVM perhaps).

The relevant comments (as well as explaining what LLVM does) are all available in `std::f32::floor` now